Dockerfile

```diff
@@ -19,8 +19,12 @@ FROM continuumio/miniconda3:4.7.12
 # Upgrade all packages to meet security criteria
 RUN apt-get update && apt-get upgrade -y && rm -rf /var/lib/apt/lists/*
 
-WORKDIR /workspace
-RUN mkdir assets
+
+RUN useradd --create-home max
+WORKDIR /home/max
```

Comment:
Many of our models have hardcoded this to `/workspace`. There might be some surprise later

Reply:
Hardcoded in places other than the Dockerfile?

Reply:
Some places reset this, some other places directly uses `/workspace`: e.g.,  https://github.com/IBM/MAX-Image-Resolution-Enhancer/blob/master/Dockerfile#L22

Reply:
I understand. I was planning on cutting a new `max-base` release after this PR is merged and then updating each model (see https://github.com/IBM/MAX-Image-Segmenter/pull/82).

We can discuss this more at the scrum tomorrow.